### PR TITLE
get default overpass query from the server and set value of textarea

### LINKED ISSUE #1838: Overpass pull dialog should allow editing of default oql
Currently the text is disabled (grayed out) until one of the building or highway presets is checked.

### PR DESCRIPTION
**Pull request by brianhatchl**
in addition to placeholder for easier editing
closes #1838

### CODE REVIEW

File: modules/Hoot/managers/api.js [hunk 1]
```diff
@@ -146,6 +147,22 @@ export default class API {
...
+    getDefaultOverpassQuery() {
+        if ( this.defaultOverpassQuery) {
+            return Promise.resolve( this.defaultOverpassQuery );
+        } else {
+            const params = {
+                path: '/grail/getDefaultOverpassQuery',
+                method: 'GET'
+            };
+            let that = this;
+            return this.request( params ).then( resp => {
```
Comment (bobi7) [inline modules/Hoot/managers/api.js]: `.then(...` on new line for syntax consistency with rest of our codebase

File: modules/Hoot/tools/overpassQueryPanel.js [hunk 2]
```diff
@@ -58,14 +58,16 @@ export default class OverpassQueryPanel {
...
-    overpassQueryPanel() {
+    async overpassQueryPanel() {
         // construct input section for user custom overpass queries
         this.overpassQueryContainer = this.form
             .select( '.wrapper div' )
             .insert( 'div', '.modal-footer' )
             .classed( 'button-wrap user-input', true );
 
-        let overpassQueryValue = '',
+        const defaultValue = await Hoot.api.getDefaultOverpassQuery();
```
Comment (bobi7) [inline modules/Hoot/tools/overpassQueryPanel.js:68]: how come we need defaultValue and overpassQueryValue?

Comment (brianhatchl) [reply modules/Hoot/tools/overpassQueryPanel.js:68]: Once you written something custom in the text area overpassQueryValue becomes that.  The placeholder will stay the default value.  I've found if you make an edit and can ctrl+z undo it, you can close the pull dialog and the default query will be restored as the editable value.

File: modules/Hoot/managers/api.js (hunk 1 above)
Comment (brianhatchl) [reply modules/Hoot/managers/api.js]: as you wish

**Review by bobi7 — APPROVED** [review]

**Pull request merged**